client/my-sites/promote-post-i2/components/campaign-item/index.tsx

```diff
@@ -44,12 +45,13 @@ export default function CampaignItem( props: Props ) {
 		status,
 		end_date,
 		budget_cents,
-		impressions_total,
-		clicks_total,
 		spent_budget_cents,
 		start_date,
+		campaign_stats,
```

Comment:
It is giving an error now because the backend is not returning the campaign_stats object.
We can add a default value so it doesn't give that error, for example:
```
campaign_stats = { impressions_total: 0, clicks_total: 0 },
```